addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -94,3 +102,81 @@ def test_edi_json(self):
             }
         }
         self.assertDictEqual(json_value, expected, "Indian EDI send json value is not matched")
+
+        #=================================== Full discount test =====================================
+        json_value = self.env["account.edi.format"]._l10n_in_edi_generate_invoice_json(self.invoice_full_discount)
+        expected = {
```

Comment:
Can not we reuse the expected variable from the previous test and just change what is changed for the 100% discount?  @jva-odoo

Reply:
Yes, good idea